app/models/report.rb

```diff
@@ -141,6 +141,12 @@ class << self
         proc: lambda { |doc| doc.preservation_size },
         solr_fields: [SolrDocument::FIELD_PRESERVATION_SIZE],
         sort: false, default: true, width: 50, download_default: false
+      },
+      {
+        field: :dissertation_id, label: 'Dissertation ID',
+        proc: lambda { |doc| doc[:identifier_ssim].map { |id| id.include?('dissertationid') ? id.split(/:/).last : nil }.join },
```

Comment:
Note, the final `join` is required here because later in the code any array is joined by `;` characters, leaving the output here like `00000001234;;` so flattening into a string avoids the appending of `;` characters.

Reply:
use `filter { |id| id.include?('dissertationid')}.map { |id| id.split(/:/).last }` and then you won't have to `join`.